crates/aptos/src/node/mod.rs

```diff
@@ -447,17 +455,152 @@ pub struct ShowValidatorSet {
 }
 
 #[async_trait]
-impl CliCommand<serde_json::Value> for ShowValidatorSet {
+impl CliCommand<ValidatorSetSummary> for ShowValidatorSet {
     fn command_name(&self) -> &'static str {
         "ShowValidatorSet"
     }
 
-    async fn execute(mut self) -> CliTypedResult<serde_json::Value> {
+    async fn execute(mut self) -> CliTypedResult<ValidatorSetSummary> {
         let client = self.rest_options.client(&self.profile_options.profile)?;
-        let response = client
-            .get_resource(CORE_CODE_ADDRESS, "0x1::stake::ValidatorSet")
-            .await?;
-        Ok(response.into_inner())
+        let validator_set: ValidatorSet = client
+            .get_account_resource_bcs(CORE_CODE_ADDRESS, "0x1::stake::ValidatorSet")
+            .await?
+            .into_inner();
+
+        ValidatorSetSummary::try_from(&validator_set)
+            .map_err(|err| CliError::BCS("Validator Set", err))
+    }
+}
+
+#[derive(Clone, Debug, Eq, PartialEq, Serialize)]
+pub struct ValidatorSetSummary {
```

Comment:
we have the same struct here https://github.com/aptos-labs/aptos-core/blob/7c0b9fc366d8ac4596d6f07d9b42630586b9be8d/types/src/on_chain_config/validator_set.rs#L25

is there a reason you want to re-implement them?

Reply:
Those structs don't show the network addresses.  This PR converts them to readable versions